config/config.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"flag"
+
+	"github.com/caarlos0/env/v8"
+)
+
+type Config struct {
+	RunAddr     string `env:"SERVER_ADDRESS"`
+	ShortenAddr string `env:"BASE_URL"`
+}
+
+func GetConfig() Config {
+
+	Cfg := Config{}
```

Comment:
```suggestion
	cfg := Config{}
```
первая большая буква в go используется только для экспорта